### PR TITLE
[WIP]Disabling FLR and SLR has explicit apis in v6

### PR DESCRIPTION
**Pull request by andreasohlund**
Waiting for https://github.com/Particular/NServiceBus/pull/3814 to be merged.

### CODE REVIEW

File: Snippets/Core/Core_6/Errors/SecondLevel/DisableWithCode.cs [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
     using NServiceBus;
-    using NServiceBus.Features;
 
     class DisableWithCode
     {
         DisableWithCode(EndpointConfiguration endpointConfiguration)
         {
             #region DisableSlrWithCode
-            endpointConfiguration.DisableFeature<SecondLevelRetries>();
+            endpointConfiguration.SecondLevelRetries().Disable();
```
Comment (indualagarsamy) [inline Snippets/Core/Core_6/Errors/SecondLevel/DisableWithCode.cs]: @Particular/nservicebus-maintainers 
Why is this inconsistent?
i.e. for FLR it's

> endpointConfiguration.DisableFirstLevelRetries();

Also what happens when you disable FLR but not SLR?

File: Snippets/Core/Core_6/UpgradeGuides/5to6/DisableRetries.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    using NServiceBus;
+
+    public class DisableRetries
+    {
+        DisableRetries(EndpointConfiguration endpointConfiguration)
+        {
+            #region 5to6-DisableRetries
+
+            endpointConfiguration.DisableFirstLevelRetries();
+            endpointConfiguration.SecondLevelRetries().Disable();
```
Comment (indualagarsamy) [inline Snippets/Core/Core_6/UpgradeGuides/5to6/DisableRetries.cs]: It just seems odd that for disabling SLR and FLR, the API is completely different.

File: Snippets/Core/Core_6/Errors/SecondLevel/DisableWithCode.cs (hunk 1 above)
Comment (timbussmann) [inline Snippets/Core/Core_6/Errors/SecondLevel/DisableWithCode.cs]: @indualagarsamy I raised a PR to change this API here: https://github.com/Particular/NServiceBus/pull/3828 please let me know what you think about my proposal

**Comment by timbussmann** [comment]
@andreasohlund I think https://github.com/Particular/docs.particular.net/pull/1608 replaces this PR, are you good with closing this one?

**Pull request closed without merging**